dbos/_migration.py

```diff
@@ -215,11 +215,25 @@ def get_dbos_migration_three(schema: str) -> str:
 """
 
 
+def get_dbos_migration_four(schema: str) -> str:
+    return f"""
+ALTER TABLE \"{schema}\".workflow_status ADD COLUMN forked_from TEXT;
+"""
+
+
+def get_dbos_migration_five(schema: str) -> str:
+    return f"""
+ALTER TABLE \"{schema}\".operation_outputs ADD COLUMN started_at_epoch_ms BIGINT, ADD COLUMN completed_at_epoch_ms BIGINT;
+"""
```

Comment:
Why not a single migration? Not that it changes much but curious about the rationale.